src/main/app/case/case.ts

```diff
@@ -140,6 +140,16 @@ export interface Case {
   childrenFirstNameAfterAdoption?: string;
   childrenLastNameAfterAdoption?: string;
 
+  otherParentAddressKnown?: YesOrNo;
+  otherParentFirstNames?: string;
+  otherParentLastNames?: string;
+  otherParentExists?: YesOrNo;
```

Comment:
Please align   otherParentExists?: YesOrNo; variable name with  birthFatherStillAlive/birthMotherStillAlive?: 

also check possible option is YesOrNo or YesOrNoOrNotSure;

@abidfs @dilipsamrahmcts @renJosephHMCTS

Reply:
checked this and currently it is just YesOrNo - with regards to the otherParentExists variable name, this was done because this question is asking whether they exist rather than whether they are still alive